src/main/resources/hudson/plugins/git/extensions/impl/GitLFSPull/config.groovy

```diff
@@ -0,0 +1,10 @@
+package hudson.plugins.git.extensions.impl.GitLFSPull;
```

Comment:
Semicolons are optional in Groovy and normally are not used.
Therefore I suggest to remove it.
(Yes, there many examples in the code that aren't good examples.)